implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -94,6 +94,17 @@ private Converters() {}
         }
     });
 
+    @SuppressWarnings("unchecked")
+    static final Converter<Integer> CHARACTER_CONVERTER = BuiltInConverter.of(11, (Converter & Serializable) value -> {
```

Comment:
Should this not be `Converter<Character>`?

Reply:
you're right